Gruntfile.js

```diff
@@ -9,9 +9,9 @@ module.exports = function(grunt) {
 
 
   // This loads all grunt tasks matching the grunt-*, @*/grunt-* patterns.
-  require('load-grunt-tasks')(grunt);
+  require('load-grunt-tasks')(grunt); //eslint-disable-line import/no-extraneous-dependencies
```

Comment:
I would probably prefer to put all the development related files that are not in the tests dir into the .eslintrc file as file glob patterns, instead of ignoring the issues line by line, e.g.:

```json
`
...
"import/no-extraneous-dependencies": [2, {
   "devDependencies": ["Gruntfile.js", "webpack.*.js", "tasks/*.js"]
}],
...
```

but let's see what @kumar303 thinks about it before applying this change.